hack/jenkins/windows_integration_test_hyperv.ps1

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 mkdir -p out
-gsutil.cmd -m cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/* out/
-cp -r out/testdata ./
+gsutil.cmd cp gs://minikube-builds/${MINIKUBE_LOCATION}/minikube-windows-amd64.exe out/
+gsutil.cmd cp gs://minikube-builds/${MINIKUBE_LOCATION}/e2e-windows-amd64.exe out/
+gsutil.cmd cp -r gs://minikube-builds/${MINIKUBE_LOCATION}/testdata .
 
-
-./out/minikube-windows-amd64.exe delete # || true
+./out/minikube-windows-amd64.exe delete
```

Comment:
why get rid of the `|| true`?  Wont this fail if for some reason the VM doesn't exist (which it really shouldn't?)